common/src/main/java/com/gentics/mesh/router/RouterStorage.java

```diff
@@ -43,6 +44,7 @@
  * Project routers are automatically bound to all projects. This way only a single node verticle is needed to handle all project requests.
  * 
  */
+@Singleton
```

Comment:
```suggestion
```
Be aware that adding `singleton` will affect how handlers will be used across verticles. In general it is bad practice to re-use handlers across verticles/threads. Most Vert.x handlers are not written with thread safety in mind.